cmd/internal/codegen/parse/parser.go

```diff
@@ -62,11 +65,53 @@ func NewAPIs(context *generator.Context, arguments *args.GeneratorArgs) *APIs {
 	b.parseControllers()
 	b.parseRBAC()
 	b.parseInformers()
+	b.checkRbac()
 	b.parseAPIs()
 	b.parseCRDs()
 	return b
 }
 
+func (b *APIs) checkRbac() {
```

Comment:
Rename it to reflect that it is validating RBAC annotations are matching Informers annotation ?
A few ugly suggestions :) 
`validateAnnotations`
`validateRBACandInformersAnnotations`
`EnsureRBACmatchesInformerAnnotations`

Reply:
Named it `validateRBACAnnotations`